pkg/chains/formats/slsa/v2alpha2/internal/resolved_dependencies/resolved_dependencies_test.go

```diff
@@ -0,0 +1,511 @@
+						{
+							PipelineResourceBinding: v1beta1.PipelineResourceBinding{ //nolint:all //incompatible with pipelines v0.45
+								Name: "nil-resource-spec",
+							},
+						}, {
+							PipelineResourceBinding: v1beta1.PipelineResourceBinding{ //nolint:all //incompatible with pipelines v0.45
+								Name: "repo",
+								ResourceSpec: &v1alpha1.PipelineResourceSpec{ //nolint:all //incompatible with pipelines v0.45
+									Params: []v1alpha1.ResourceParam{ //nolint:all //incompatible with pipelines v0.45
+										{Name: "url", Value: "https://github.com/GoogleContainerTools/distroless"},
+									},
+									Type: backport.PipelineResourceTypeGit,
+								},
+							},
+						},
+					},
+				},
+			},
+			Status: v1beta1.TaskRunStatus{
+				TaskRunStatusFields: v1beta1.TaskRunStatusFields{
+					TaskRunResults: []v1beta1.TaskRunResult{
+						{
+							Name: "img1_input" + "-" + artifacts.ArtifactsInputsResultName,
+							Value: *v1beta1.NewObject(map[string]string{
+								"uri":    "gcr.io/foo/bar",
+								"digest": digest,
+							}),
+						},
+					},
+					ResourcesResult: []v1beta1.PipelineResourceResult{
+						{
+							ResourceName: "repo",
+							Key:          "commit",
+							Value:        "50c56a48cfb3a5a80fa36ed91c739bdac8381cbe",
+						}, {
+							ResourceName: "repo",
+							Key:          "url",
+							Value:        "https://github.com/GoogleContainerTools/distroless",
+						},
+					},
+				},
+			},
+		},
+		want: []*v1.ResourceDescriptor{
+			&v1.ResourceDescriptor{
+				Name: "inputs/result",
+				URI:  "gcr.io/foo/bar",
+				Digest: common.DigestSet{
+					"sha256": strings.TrimPrefix(digest, "sha256:"),
+				},
+			},
+			&v1.ResourceDescriptor{
+				Name: "pipelineResource",
+				URI:  "git+https://github.com/GoogleContainerTools/distroless.git",
+				Digest: common.DigestSet{
+					"sha1": "50c56a48cfb3a5a80fa36ed91c739bdac8381cbe",
+				},
+			},
+		},
+	}, {
+		name: "resolvedDependencies from remote task",
+		taskRun: &v1beta1.TaskRun{
+			Status: v1beta1.TaskRunStatus{
+				TaskRunStatusFields: v1beta1.TaskRunStatusFields{
+					Provenance: &v1beta1.Provenance{
+						RefSource: &v1beta1.RefSource{
+							URI: "git+github.com/something.git",
+							Digest: map[string]string{
+								"sha1": "abcd1234",
+							},
+						},
+					},
+				},
+			},
+		},
+		want: []*v1.ResourceDescriptor{
+			&v1.ResourceDescriptor{
+				Name: "task",
+				URI:  "git+github.com/something.git",
+				Digest: common.DigestSet{
+					"sha1": "abcd1234",
+				},
+			},
+		},
+	}, {
+		name: "resolvedDependencies from git results",
+		taskRun: &v1beta1.TaskRun{
+			Spec: v1beta1.TaskRunSpec{
+				Params: []v1beta1.Param{{
+					Name:  "CHAINS-GIT_COMMIT",
+					Value: *v1beta1.NewStructuredValues("my-commit"),
+				}, {
+					Name:  "CHAINS-GIT_URL",
+					Value: *v1beta1.NewStructuredValues("github.com/something"),
+				}},
+			},
```

Comment:
this seems to test resolving the deps from params instead of results, but the test case name is called `... git results`.

Reply:
renamed it to `git resolved dependencies from taskrun params`